setup.py

```diff
@@ -80,7 +80,7 @@ def finalize_options(self):
     install_requires=requirements,
     extras_require={
         "docs": ["sphinx", "sphinx_rtd_theme", "recommonmark", "sphinxcontrib-bibtex", "sphinx_markdown_tables"],
-        "tests": ["cirq"],
+        "tests": ["cirq", "ply"],
```

Comment:
I noticed that it is required to install `ply` in order to run these tests but this is due to a `cirq` bug right? Shouldn't this be in Cirq's requirements?

Reply:
I think so, on my laptop installing cirq seems to be sufficient, however the github actions are failing without this adding this dependency explicitly.